service/src/main/java/com/codeforcommunity/dataaccess/AuthDatabaseOperations.java

```diff
@@ -21,6 +23,7 @@
 
     private final DSLContext db;
     private AuthUtils sha;
+    public static final int CUTOFF_TIME = 86400;
```

Comment:
This should be moved to a config file (see the property file loader in common) and also named something that indicates the units you're using

Reply:
sure, do we have a config file already?